build.gradle

```diff
@@ -4,6 +4,8 @@ buildscript {
 }
 
 plugins {
+    id 'maven-publish'
```

Comment:
Hmm I think this shouldn't be needed? The `alias libs.plugins.mavenPublish apply false` adds the plugin without applying, so each plugin needs to apply it later

Reply:
Maybe not, but we're not merging this PR anyway.